app/src/main/res/values/strings.xml

```diff
@@ -52,9 +52,13 @@
     <string name="menu_page_other_languages">Change language</string>
     <string name="menu_page_find_in_page">Find in article</string>
     <string name="menu_page_add_to_watchlist">Add to Watchlist</string>
+    <string name="menu_page_add_to_watchlist_v2">Watch</string>
     <string name="menu_page_view_talk">View talk page</string>
+    <string name="menu_page_view_talk_v2">Talk page</string>
     <string name="menu_page_view_edit_history">View edit history</string>
+    <string name="menu_page_view_edit_history_v2">Edit history</string>
     <string name="menu_page_remove_from_watchlist">Remove from Watchlist</string>
+    <string name="menu_page_remove_from_watchlist_v2">Watched</string>
```

Comment:
can we name these strings as menu_page_watched, menu_page_talk,menu_page_watch, and so on instead of v2?

Reply:
Make sense, done!